laceworksdk/http_session.py

```diff
@@ -96,15 +100,18 @@ def _check_access_token(self):
         A method to check the validity of the access token.
         """
 
-        if self._access_token is None or self._access_token_expiry < datetime.now(timezone.utc):
-
+        if self._access_token is None or self._access_token_expiry < datetime.now(
+            timezone.utc
+        ):
             response = self._get_access_token()
 
             # Parse and restructure the returned date (necessary for Python 3.6)
             expiry_date = response.json()["expiresAt"].replace("Z", "+0000")
 
             # Update the access token and expiration
-            self._access_token_expiry = datetime.strptime(expiry_date, "%Y-%m-%dT%H:%M:%S.%f%z")
+            self._access_token_expiry = datetime.strptime(
+                expiry_date, "%Y-%m-%dT%H:%M:%S.%f%z"
+            )
             self._access_token = response.json()["token"]
 
     def _check_response_code(self, response, expected_response_codes):
```

Comment:
You show the return type, but why are we missing the `-> str` type hint in the code?